### PR TITLE
Add tsdoc for IFluidContainer

### PR DESCRIPTION
**Pull request by ChumpChief**

### CODE REVIEW

File: api-report/fluid-static.api.md [hunk 1]
```diff
@@ -41,22 +41,17 @@ export class DOProviderContainerRuntimeFactory extends BaseContainerRuntimeFacto
     protected containerInitializingFirstTime(runtime: IContainerRuntime): Promise<void>;
     }
 
-// @public (undocumented)
+// @public
```
Comment (skylerjokiel) [inline api-report/fluid-static.api.md:44]:
```suggestion
/**
 * @public
 */
```

Comment (skylerjokiel) [reply api-report/fluid-static.api.md:44]: Oh snap... wasn't paying attention to the file... Ignore this.

File: packages/framework/fluid-static/src/fluidContainer.ts [hunk 2]
```diff
@@ -10,18 +10,50 @@ import { AttachState } from "@fluidframework/container-definitions";
...
+    /**
+     * Create a new data object or DDS of the specified type.  In order to share the data object or DDS with other
+     * collaborators and retrieve it later, store its handle in a collection like a SharedDirectory from your
+     * initialObjects.
+     * @param objectClass - The class of data object or DDS to create
+     */
     create<T extends IFluidLoadable>(objectClass: LoadableObjectClass<T>): Promise<T>;
+
+    /**
+     * Dispose of the container, permanently disabling it.
```
Comment (skylerjokiel) [inline packages/framework/fluid-static/src/fluidContainer.ts:49]:
```suggestion
     * Dispose of the container instance, permanently disabling it.
```

File: packages/framework/fluid-static/src/fluidContainer.ts [hunk 3]
```diff
@@ -10,18 +10,50 @@ import { AttachState } from "@fluidframework/container-definitions";
...
+     */
     readonly connected: boolean;
+
+    /**
+     * Whether the container is disposed, which permanently disables it.
+     */
     readonly disposed: boolean;
+
+    /**
+     * The collection of data objects and DDSs that were specified by the schema used to create or load the container.
```
Comment (skylerjokiel) [inline packages/framework/fluid-static/src/fluidContainer.ts:36]:
```suggestion
     * The collection of data objects and DDSes that were specified by the schema. These data objects and DDSes exist for the lifetime of the container.
```

**Review by tylerbutler — APPROVED** [review]
I only looked at the API report changes.

File: packages/framework/fluid-static/src/fluidContainer.ts [hunk 4]
```diff
@@ -10,18 +10,50 @@ import { AttachState } from "@fluidframework/container-definitions";
...
 export interface IFluidContainerEvents extends IEvent {
     (event: "connected" | "dispose" | "disconnected", listener: () => void): void;
 }
 
+/**
+ * The IFluidContainer provides an entrypoint into the client side of collaborative Fluid data.  It provides access
+ * to the data as well as status on the collaboration session.
+ */
 export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
+    /**
```
Comment (skylerjokiel) [inline packages/framework/fluid-static/src/fluidContainer.ts:25]: Looks like `attach()` wasn't added to the interface. Can you add it here.

File: packages/framework/fluid-static/src/fluidContainer.ts [hunk 5]
```diff
@@ -10,18 +10,50 @@ import { AttachState } from "@fluidframework/container-definitions";
 import { LoadableObjectClass, LoadableObjectRecord } from "./types";
 import { RootDataObject } from "./rootDataObject";
 
+/**
+ * Events emitted from IFluidContainer.
```
Comment (skylerjokiel) [inline packages/framework/fluid-static/src/fluidContainer.ts:14]: We really need to figure out how to document this syntax in a consumable way.

File: packages/framework/fluid-static/src/fluidContainer.ts (hunk 4 above)
Comment (ChumpChief) [reply packages/framework/fluid-static/src/fluidContainer.ts:25]: Got it prepped in another branch, was gonna put it up immediately following this change.

**Review by skylerjokiel — APPROVED** [review]
can you add the `attach()` api to the IFluidContainer interface.

File: packages/framework/fluid-static/src/fluidContainer.ts (hunk 5 above)
Comment (skylerjokiel) [reply packages/framework/fluid-static/src/fluidContainer.ts:14]: Don't block on this but it's definitely a sore spot in our API.

**Review by skylerjokiel — APPROVED** [review]
🍁

**Pull request merged**